### ISSUE TITLE
Απορια για το 1.6 

### ISSUE THREAD

**Issue by mpektkd** [ntua-cvsp-lab/DSP_LabSupport#35]
Καλησπερα. Οσο αφορα το 1.6 πρεπει η αντιστοιχιση να γινει αποκλειστικα με βαση τις συχνοτητες διοτι λογω της οχι τελειας ακριβειας των bits η αποθηκευση των δειγματων μου γινεται με διαφορετικο τροπο κι οι συχνοτητες που μου προκυπτουν απο την ευρεση των k ειναι προσεγγιστικα πολυ κοντα με τις touch tone αλλα οχι ισες.Ειναι λαθος η συγκριση να γινει με τα k τα οποια ειναι ακεραια και δεν χανεται η ακριβεια.Ευχαριστω πολυ.

**Comment by cgaroufis** [comment]
Σωστά, η σύγκριση θα πρέπει όντως να γίνει με βάση τους ακέραιους δείκτες k τους οποίους έχετε βρει για το κάθε ψηφίο (είτε εποπτικά είτε θεωρητικά) από το ερώτημα 1.5.

Καλή συνέχεια,
Χρστος
